azurelinuxagent/common/osutil/freebsd.py

```diff
@@ -93,6 +97,192 @@ def get_if_mac(self, ifname):
     def get_first_if(self):
         return self._get_net_info()[:2]
 
+    @staticmethod
+    def read_route_table():
+        """
+        Return a list of strings comprising the route table, including column headers. Each line is stripped of leading
+        or trailing whitespace but is otherwise unmolested.
+
+        :return: Entries in the route priority table from `netstat -rn`
+        :rtype: list(str)
+        """
```

Comment:
Could you please add an example of how the `netstat -rn` output differs with the linux based `proc/net/route` header just to make sure what this function is parsing

Reply:
Exmples added as requested in e9d7a64